macros/clone_schema.sql

```diff
@@ -253,15 +281,26 @@
             Because of this effect we are applying a COPY GRANTS syntax to single objects grant statements which are generated in loop.
         */#}
 
+        {#/*
+            This block of code is intended to catching of not correct values for the schemas' names.
```

Comment:
Nitpick: Can improve the statement as "This block of code is intended to catch incorrect values for the schemas' names." Although "invalid" might be better than "incorrect" here.

Reply:
Thank you, Chris! I've corrected this comment.